galaxy_importer/loaders.py

```diff
@@ -281,7 +283,29 @@ def _get_readme(self):
 
     def _get_metadata_description(self):
         self.log.info('Getting role description')
-        return ''
+        description = None
+        metadata_file = self._get_metadata_file(self.rel_path)
+
+        if not metadata_file:
+            self.log.warning('No role metadata found')
+            return description
+
+        with open(metadata_file) as fp:
+            role_metadata = yaml.safe_load(fp)
```

Comment:
Could maybe use some hardening if yaml.safe_load() raises exceptions.

Reply:
agreed, i will add this